migrations/sql/V2020.08.04.1450__AddSnapshotDataset.sql

```diff
@@ -0,0 +1,7 @@
+CREATE TABLE snapshot_dataset
+(
+    snapshot TEXT PRIMARY KEY references version_snapshot (id),
+    id TEXT,
+    name TEXT,
+    last_modified TEXT
```

Comment:
You don't want to use a timestamp for this?

Reply:
Actually I think it needs to be `revision_id`, datasets don't have a `last_modified`